### PR TITLE
Page Replacement Algorithms PR

### PR DESCRIPTION
**Pull request by shraiyya**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

PR FOR ISSUE #2054 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

**Please add DWOC label and level.**

### CODE REVIEW

File: Java/Operating-System/allocate.java [hunk 1]
```diff
@@ -0,0 +1,641 @@
...
+Privacy
+Security
+Status
+Docs
+Contact GitHub
+Pricing
+API
+Training
+Blog
+About
```
Comment (HarshCasper) [inline Java/Operating-System/allocate.java]: Please remove this. It looks copy-pasted

File: Java/Operating-System/allocate.java [hunk 2]
```diff
@@ -0,0 +1,641 @@
...
+
+*/
+import java.util.*;
+
+class node
+{
+	int d;
+	node next;
+
+	node() //constructor of node class
```
Comment (HarshCasper) [inline Java/Operating-System/allocate.java]: Use comments above the Line of Code. Follow the same pattern across the rest of your code

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few more changes in general: 

- Remove the extra lines of code which are redundant
- Put Comments above the Line of Code
- Run Code formatted to format the Code

**Comment by shraiyya** [comment]
I have done the changes.
Please review and add the tag of level of DWOC.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**